src/_xoConnection.js

```diff
@@ -131,6 +137,34 @@ class XoConnection extends Xo {
     return id;
   }
 
+  async runBackupJob(
```

Comment:
Tell me if this method is not needed (I will put all the next code in `backupNg.spec.js`).
I did this helper in case other scenarios need it.

Reply:
I suggest: 
```js
runBackupJob(jobId, scheduleId, modes, nExecutions)
```